streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -246,7 +246,7 @@ default boolean commitRequested() {
 
     // IQ related methods
 
-    StateStore getStore(final String name);
+    StateStore store(final String name);
```

Comment:
Should we rename `StateManager#getStore(String)` to `StateManager#store(String)`

Reply:
Yes, but I will do this in a follow up PR to keep the scope of individual PRs small.

Reply:
Here we go: https://github.com/apache/kafka/pull/17025